libraries/botbuilder-azure-blobs/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "botbuilder-azure-blobs",
+  "author": "Microsoft Corp.",
+  "description": "Botbuild storage bindings for Azure Blobs services",
+  "version": "4.1.6",
+  "license": "MIT",
+  "keywords": [
+    "botbuilder",
+    "botframework",
+    "bots",
+    "chatbots"
+  ],
+  "bugs": {
+    "url": "https://github.com/Microsoft/botbuilder-js/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/botbuilder-js.git"
+  },
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "typesVersions": {
+    "<3.9": {
+      "*": [
+        "_ts3.4/*"
+      ]
+    }
+  },
+  "dependencies": {
+    "@azure/storage-blob": "^12.2.1",
+    "botbuilder": "4.1.6",
+    "get-stream": "^6.0.0",
+    "p-map": "^4.0.0"
+  },
+  "devDependencies": {
+    "@microsoft/api-extractor": "^7.10.4",
+    "downlevel-dts": "^0.6.0",
+    "mocha": "^8.1.3",
+    "nyc": "^15.1.0",
+    "sinon": "^9.2.0",
+    "typedoc": "^0.19.2",
+    "typescript": "^4.0.3"
+  },
+  "scripts": {
+    "test": "npm run build && nyc mocha tests",
```

Comment:
could you add `--check-leaks` to the mocha call? We should be running this in every package.json, but let's get this new package setup nicely from the start